corehq/apps/translations/integrations/transifex/project_migrator.py

```diff
@@ -82,6 +62,30 @@ def _update_slugs(self):
             responses[old_id] = self.client.update_resource_slug(resource_slug, new_resource_slug)
         return responses
 
+    @memoized
+    def _get_slug_prefix(self, resource_type):
+        return TRANSIFEX_SLUG_PREFIX_MAPPING.get(resource_type)
+
+    def _update_menus_and_forms_sheet(self):
+        langs = copy.copy(self.source_app_langs)
+        translations = OrderedDict()
+        for lang in langs:
+            try:
+                translations[lang] = self.client.get_translation("Menus_and_forms", lang, lock_resource=False)
+            except ResourceMissing:
+                # Probably a lang in app not present on Transifex, so skip
```

Comment:
Not quite sure of the background of what Transfix migration involves, but should a user (or whoever is migrating) be warned that a language is not missing in the new project by not losing this exception?

Reply:
It's possible that a language in added to the app but has not yet made available on transifex. Its not something a user would miss :)
and fine to skip here coz,
If its not present on Transifex then it means there are no translations that need to be migrated.